src/modules/fs_archive_events.js

```diff
@@ -0,0 +1,37 @@
+const { __ } = wp.i18n;
+const { registerBlockType } = wp.blocks;
+function bob_ross23() {
+	return 'bob_ross23';
+}
+console.log(bob_ross23());
+registerBlockType( 'tec/archive-events', {
```

Comment:
We want to use `tec/` 'theme' namespace here?

Reply:
Yes, we're moving away from `tribe`